### ISSUE TITLE
Warning when using 1080p50

### ISSUE THREAD

**Issue by dotarmin** [CasparCG/server#1299]
### Expected behaviour
Use 1080p50 as video mode without getting warnings.

### Current behaviour
When set to 1080p5000, warnings are printed in the console and the log: `Device does not support video-format: 1080p50`. Output is visible on a monitor in 1080p50.

```
[2020-05-20 09:01:41.496] [info]    Initialized OpenGL Accelerated GPU Image Mixer for channel 1
[2020-05-20 09:01:41.497] [info]    video_channel[1|1080p5000] Successfully Initialized.
[2020-05-20 09:01:41.497] [info]    Initialized OpenGL Accelerated GPU Image Mixer for channel 2
[2020-05-20 09:01:41.497] [info]    video_channel[2|1080p5000] Successfully Initialized.
[2020-05-20 09:01:41.497] [info]    Initialized OpenGL Accelerated GPU Image Mixer for channel 3
[2020-05-20 09:01:41.497] [info]    video_channel[3|1080p5000] Successfully Initialized.
[2020-05-20 09:01:41.497] [info]    Initialized OpenGL Accelerated GPU Image Mixer for channel 4
[2020-05-20 09:01:41.497] [info]    video_channel[4|1080p5000] Successfully Initialized.
[2020-05-20 09:01:41.498] [info]    Initialized OpenGL Accelerated GPU Image Mixer for channel 5
[2020-05-20 09:01:41.498] [info]    video_channel[5|1080p5000] Successfully Initialized.
[2020-05-20 09:01:41.528] [warning] Device does not support video-format: 1080p50
[2020-05-20 09:01:41.529] [info]    DeckLink Duo 2 [1-1|1080p5000] Enabled embedded-audio.
[2020-05-20 09:01:41.529] [info]    DeckLink Duo 2 [1-1|1080p5000] Disabled low-latency mode.
[2020-05-20 09:01:41.529] [error]   DeckLink Duo 2 [1-1|1080p5000] Failed to enable external keyer.
[2020-05-20 09:01:41.548] [info]    DeckLink Duo 2 [1-1|1080p5000] Initialized.
[2020-05-20 09:01:41.568] [warning] Device does not support video-format: 1080p50
[2020-05-20 09:01:41.568] [info]    DeckLink Duo 2 [2-2|1080p5000] Enabled embedded-audio.
[2020-05-20 09:01:41.568] [info]    DeckLink Duo 2 [2-2|1080p5000] Disabled low-latency mode.
[2020-05-20 09:01:41.568] [error]   DeckLink Duo 2 [2-2|1080p5000] Failed to enable external keyer.
[2020-05-20 09:01:41.587] [info]    DeckLink Duo 2 [2-2|1080p5000] Initialized.
[2020-05-20 09:01:41.589] [info]    DeckLink Duo 2 [1-1|1080p5000] Reference signal: not detected.
[2020-05-20 09:01:41.615] [warning] Device does not support video-format: 1080p50
[2020-05-20 09:01:41.616] [info]    DeckLink Duo 2 [3-3|1080p5000] Enabled embedded-audio.
[2020-05-20 09:01:41.616] [info]    DeckLink Duo 2 [3-3|1080p5000] Disabled low-latency mode.
[2020-05-20 09:01:41.616] [error]   DeckLink Duo 2 [3-3|1080p5000] Failed to enable external keyer.
[2020-05-20 09:01:41.628] [info]    DeckLink Duo 2 [2-2|1080p5000] Reference signal: not detected.
[2020-05-20 09:01:41.636] [info]    DeckLink Duo 2 [3-3|1080p5000] Initialized.
```

----

### Steps to reproduce
1. Set your server to run in `1080p5000`
2. Start the server

### Environment
* Server version: CasparCG Server v2.3.0 LTS Beta 1
* Operating system: Windows 10
* Decklink: Duo 2

**Comment by Julusian** [comment]
This isnt a new issue, Ive been seeing this with a Mini-Monitor with 1080i50 for years.
Perhaps the simple solution is to change that to an info line saying `Device might not support video-format: 1080p50`? 

I suspect that updating the minimum decklink driver version might help, but I know that many users have their preferred version so we will need to be careful.

**Comment by Julusian** [comment]
Fixed in 862e005b
